x/mongo/driver/operation.go

```diff
@@ -399,6 +401,7 @@ func (op Operation) Execute(ctx context.Context, scratch []byte) error {
 		op.cmdName = startedInfo.cmdName
 		startedInfo.redacted = op.redactCommand(startedInfo.cmdName, startedInfo.cmd)
 		startedInfo.serviceID = conn.Description().ServiceID
+		startedInfo.serverConnID = desc.ConnectionID
```

Comment:
```suggestion
		startedInfo.serverConnID = conn.Description().ConnectionID
```

IIUC `desc` is the server description returned by server selection. `desc.ConnectionID` is associated with the connection used in monitoring.

The event's connection ID should be associated with the connection used for the operation.

Reply:
As discussed offline, `desc.ConnectionID` and `conn.Description().ConnectionID` are equivalent because `desc.Server` should be equal to `conn.Description()`. Because wire version checks above are done with `desc.WireVersion*`, I think using `desc.ConnectionID` also makes intuitive sense.

Reply:
Thank you for the explanation.